docs/rocketchat/get-help-in-rocketchat.md

```diff
@@ -1,95 +1,99 @@
+     wildcardPolicy: None
 
-author: Alex Lloyd
+The user tells us the where, when, what, and how of an issue. We want to know what you expected, what happened, and in what environment, the problem happened. Before connecting with us, check [Jon Skeet's checklist](https://codeblog.jonskeet.uk/2012/11/24/stack-overflow-question-checklist/) if you've done everything possible before posting.
 
-editor: Pilar Solares
+Below, we offer more guidance to strengthen a question. Strong questions simplify the response time from a community member. 
 
-content_owner: Olena Mitvoska
+## The 'how' and 'what'
 
-sort_order: 4
----
+### How
 
-# Get help in Rocket.Chat
-Last updated: **July 10, 2023**
+Please tell us the problem, what you expected when doing a task, and the importance of it.
 
-To get support on Rocket.Chat, you need to ask questions. Including the right information in your questions can help you get a better answer more quickly. 
-Below, we outline the information you should include when asking for help or support from the platform community on Rocket.Chat.
+If it’s a sensitive task, please specify confidentiality is necessary. Being forthcoming about confidentiality stops us from expecting more information.  
 
-## On this page
-* [**Where has the issue ocurred?**](#where-has-the-issue-ocurred)
-* [**What are you trying to do?**](#what-are-you-trying-to-do)
-* [**What is the problem?**](#what-is-the-problem)
-* [**What have you done to fix the problem?**](#what-have-you-done-to-fix-the-problem)
-* [**What kind of help do you want?**](#what-kind-of-help-do-you-want)
-* [**Posting a question**](#posting-a-question)
-* [**Related pages**](#related-pages)
-<!-- ### End of "On this page" -->
+**Please don’t include** sensitive artifacts, like passwords, tokens, API keys, etc.
 
----
-## Where has the issue ocurred?
+### What
 
-**If the problem is with a cluster**
+1. Tell us about the nature of the problem, what it looks like and how it’s behaving. For example, if a screen doesn’t load as you expected, give us a description of what it looks like, and what it should look like
 
-In what cluster and namespace are you experiencing the problem? Provide the namespace name, so the community knows exactly where to find the problem.
+    **Please note:** Screenshots, while helpful, won’t be accessible without alt text.  
 
-**If the problem is with a tool**
+2. After detailing the problem, we want to know what troubleshooting you tried. Being proactive helps us narrow down solutions and stops repetition
 
-What tool are you trying to use? Let the community know which platform tool is causing the problem.
+    Please give us a link to the source if you followed steps on a StackOverflow thread or read documentation about the problem
 
-**If the problem is with a stack**
+3. Last, please tell us what type of help you’re looking for. A solution, link to resources, and/or another way to complete a task? Please be very clear so we don’t resort to guessing
```

Comment:
"Are you looking for a solution..."?

"don't misinterpret your need" over "resort to guessing"